integration-test.sh

```diff
@@ -13,9 +13,23 @@ readonly DISTRIBUTIONS=(2.4.6 5.6.7)
 readonly TRACKS=(geonames nyc_taxis http_logs nested)
 
 readonly ES_METRICS_STORE_VERSION="6.2.1"
+readonly MIN_CURL_VERSION=(7 12 3)
 
 ES_PID=-1
 
+function check_prerequisites() {
+    local curl_major_version=$(curl --version | head -1 | cut -d ' ' -f 2,2 | cut -d '.' -f 1,1)
+    local curl_minor_version=$(curl --version | head -1 | cut -d ' ' -f 2,2 | cut -d '.' -f 2,2)
+    local curl_patch_release=$(curl --version | head -1 | cut -d ' ' -f 2,2 | cut -d '.' -f 3,3)
+
+    if [[ $curl_major_version < ${MIN_CURL_VERSION[0]} ]] || \
+       [[ $curl_major_version == ${MIN_CURL_VERSION[0]} && $curl_minor_version < ${MIN_CURL_VERSION[1]} ]] || \
```

Comment:
Hmm, I don't think this will work anymore when curl 8.x will be released in the future? While the check is good, I am also not sure whether we can just avoid this check completely because we have tight control over the environments in which this runs (our CI)?

Reply:
It should work (I've tested it too) because we are **failing** only if:

- curl < 7 or
- curl_major == 7 and curl_minor < 12 or
- curl_major == 7 and curl_minor == 12 and curl_patch < 3

everything else passes, including curl >= 8

Reply:
Ah, I missed that. Good point.